capstone/fitness/src/main/java/com/google/sps/util/Scheduler.java

```diff
@@ -0,0 +1,71 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License. 
+package com.google.sps.util;
+
+import com.google.api.client.util.DateTime;
+import com.google.api.services.calendar.model.EventDateTime;
+import com.google.api.services.calendar.model.Event;
+import java.util.*;
+
+// class scheduler uses the user's free/busy information and the duration of an exercise to find a time in a user's day when
+// the user could do an exercise
+public class Scheduler {
+  private final long exerciseDuration;
+
+  public Scheduler(long exerciseDuration) {
+    this.exerciseDuration =exerciseDuration;
+  }
+ 
+  // minSpan and maxSpan are the boundaries for when a workout could be scheduled on a particular day e.g 7:00 AM to 9:00 PM
```

Comment:
nit: see javadoc styling for public methods

Reply:
Fixed.